content/Arduino Cloud/General Cloud/Install-the-Arduino-Create-Agent.md

```diff
@@ -3,32 +3,75 @@ title: "Install the Arduino Create Agent"
 id: 360014869820
 ---
 
-Arduino Cloud uses its own agent to be the middle man between Arduino's Web Editor and IoT Cloud and your computer. In this article we'll explain how to install it in your system (Windows, Mac, and Linux Ubuntu).
+To use the Arduino Web Editor or Arduino IoT Cloud, the **Arduino Create Agent** must be running on your computer.
+
+Learn how to install the Arduino Create Agent on Windows, macOS, and Linux computers.
+
+If you're using Chromebook, see [Use Arduino with Chromebook](https://support.arduino.cc/hc/en-us/articles/360016495639-Use-Arduino-with-Chromebook) instead.
 
 ---
 
-1. Open the [Arduino Create agent installation](https://create.arduino.cc/getting-started/plugin/welcome) page and click **Start**.
+## Install the Arduino Create Agent
+
+Follow these steps:
+
+1. Open the **[Arduino Create agent installation page](https://create.arduino.cc/getting-started/plugin/welcome)**
+
+   ![Arduino Create agent installation page](img/create-agent-install-start_600px.png)
+
+1. Click **START**.
 
-   ![Arduino Create agent installation page](img/create-agent-install-start.png)
+1. Click **DOWNLOAD** to download the installer.
 
-2. Click **Download** to download the installer. Go to your download folder, double-click the installer and follow the instructions. The installation guide will begin looking for the installed agent:
+1. Leave your web browser open, and open your downloads folder.
 
-   * If you intend to user the Create Agent with a browser other than Google Chrome or Mozilla Firefox, make sure to answer Yes in the Browser Support step.
-   * If you're installing the Create Agent for Windows, you may receive a security warning and be asked to approve installing a root certificate from Arduino. Choose Yes to install the certificate and proceed with the installation.
+1. Install the application:
 
-   ![Arduino Create agent installation with "Looking for ins agent" message highlighted"](img/create-agent-install-install-macos.png)
+   * **Windows:** Double-click the .exe installer and follow the instructions.
+     * You may receive a security warning and be asked to approve installing a root certificate from Arduino. Choose **Yes** to install the certificate and proceed with the installation.
+   * **macOS:** Double-click the .dmg file. Drag ArduinoCreateAgent.app into the Applications folder shortcut.
+   * **Linux:** Extract the .tar.gz archive and double-click the .run file. Follow the instructions.
 
-3. When it's found, you will see this page. Click **Next** to finish the installation guide.
+1. To confirm that the Create Agent is running, open your web browser with the Arduino Installation page.
 
-   ![Agent correctly installed confirmation page](img/create-agent-install-correctly.png)
```

Comment:
I would consider adding this image back (otherwise the next step might be confusing to follow)